File: midterm2/q05/person_details.scala
object SimpleTuple {
  def getPersonDetails: (String, Int, Double) = ("Alice", 30, 5.6)

  def main(args: Array[String]): Unit = {
    val (name, age, height) = getPersonDetails
    println(s"Name: $name")
    println(s"Age: $age")
    println(s"Height: $height")
  }
}

File: midterm2/q05/person_details.hs
getPersonDetails :: (String, Int, Double)
getPersonDetails = ("Alice", 30, 5.6)

main :: IO ()
main = do
  let (name, age, height) = getPersonDetails
  putStrLn $ "Name: " ++ name
  putStrLn $ "Age: " ++ show age
  putStrLn $ "Height: " ++ show height
